scripts/SANS/sans/gui_logic/presenter/add_runs_presenter.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, sum_runs_model, view, parent_view):
 
         self.save_directory = ""
         self._connect_to_view(view)
+        self._handle_custom_outfile_check_changed(False)
```

Comment:
Am I right in thinking that we're only making this call to `_handle_custom_outfile_check_changed` to disable the output filename line edit when the tab is created? The other steps that are taken in that method are either duplicating what's already been done in the `__init__` method or aren't necessary at this stage? If that's correct, I wonder if we could just call `self._view.disable_output_file_name_edit()` directly here?